frontend2/src/content/bc22.ts

```diff
@@ -0,0 +1,68 @@
+import type {
+  CommonIssuesPageKey,
+  QuickStartedPageKey,
+  ResourcesPageKey,
+  TourneyPageKey,
```

Comment:
Make sure to run `npm run lint` or `npm run format` to make sure any lint issues are fixed before committing (idk why these aren't thrown as errors by our CD pipeline though...)